File: MasterCatalog.Domain/Models/ItemAttributeValue.cs
using System.ComponentModel.DataAnnotations;

namespace MasterCatalog.Domain.Models
{
    public class ItemAttributeValue
    {
        public int? ItemAttributeValueID { get; set; }

        [DeniedValues(0, ErrorMessage = "Item Attribute identifier is required")]
        public int ItemAttributeID { get; set; }

        [DeniedValues(0, ErrorMessage = "Item identifier is required")]
        public int ItemID { get; set; }

        [Required(AllowEmptyStrings = false, ErrorMessage = "Attribute value is required")]
        public string AttributeValue { set; get; } = string.Empty;

        public int DisplayOrder { set; get; }
    }
}

File: MasterCatalog.Web/wwwroot/js/models/itemAttributeValue.js
export class ItemAttributeValue {
    itemAttributeValueID;
    itemID;
    itemAttributeID;
    attributeValue;
    displayOrder;

    constructor(itemAttributeValueID, itemID, itemAttributeID, attributeValue, displayOrder) {
        this.itemAttributeValueID = itemAttributeValueID;
        this.itemID = itemID;
        this.itemAttributeID = itemAttributeID;
        this.attributeValue = attributeValue;
        this.displayOrder = displayOrder;
    }
}